train.py

```diff
@@ -111,12 +113,15 @@ def train(params):
     # Initialize tensorboard writer
     pbar = tqdm.tqdm(data_loader)
     for i, rollout_batch in enumerate(pbar):
-        rollout_batch = rollout_batch.float().to(device)
-        error, kld, prediction = hgn.fit(rollout_batch)
+        # rollout_batch has shape (batch_len, seq_len, channels, height, width)
+        rollout_batch = rollout_batch.to(device)
+        error, kld, prediction = hgn.fit(rollout_batch, variational=variational)
         training_logger.step(losses=(error, kld),
```

Comment:
Maybe passing none to this crashes the program, did you test it?

Reply:
Yep, in `TrainingLogger` I save the kl loss only if it's not None